context.go

```diff
@@ -749,14 +755,19 @@ loop:
 			if tooManyErrors {
 				continue
 			}
-			if blueprintsSet[blueprint.string] {
+			// Limit concurrent calls to parseBlueprintFiles to 200
+			// Darwin has a default limit of 256 open files
```

Comment:
Any idea if this limits our performance? should we either auto-detect this, or have separate mac vs linux limits?